substrate/frame/scheduler/src/lib.rs

```diff
@@ -273,6 +284,15 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	#[pallet::storage]
+	pub type Retries<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
```

Comment:
Looking at it again, I think it's safe to have this be `Twox64Concat`. @ggwpez WDYT

Reply:
Its only accessed through permissioned origins - so it is trusted already. In this case it should also not easily disbalance the trie so should be be fine to use twoX. But TBH the advantage is negligible as trade-off against a more balanced trie.

Reply:
I'll just leave it as is then.